pwndbg/commands/checksec.py

```diff
@@ -18,8 +26,9 @@ def color_lines(output: str) -> str:
     return "\n".join(map(color_line, output.split("\n")))
 
 
-@pwndbg.commands.ArgparsedCommand("Prints out the binary security settings using `checksec`.")
+@pwndbg.commands.ArgparsedCommand(parser, command_name="checksec")
 @pwndbg.commands.OnlyWithFile
-def checksec() -> None:
-    output = pwndbg.wrappers.checksec.get_raw_out(pwndbg.gdblib.file.get_proc_exe_file())
+def checksec(file: str) -> None:
+    local_path = file or pwndbg.gdblib.file.get_proc_exe_file()
+    output = pwndbg.wrappers.checksec.get_raw_out(local_path)
```

Comment:
I believe we can just invoke pwnlib's ELF.checksec() here directly instead of implementing it in another file.

Reply:
I didn't use `ELF.checksec()` here because `relro_status` and `pie_status` in `wrappers/checksec.py` depend on `get_raw_out`. So I thought `get_raw_out` should not be removed.

Or I can do the same thing here and keep `get_raw_out`  in  `wrappers/checksec.py`